src/forms/lib/InputField.tsx

```diff
@@ -24,9 +24,6 @@ export const InputField = ({
       render={({ field: { onChange, value, ref } }) => (
         <Input
           style={innerStyle}
-          onChange={value => {
```

Comment:
Are you sure that it can be deleted?
I think it will affect inputs in project

Reply:
Agree. Rolled back